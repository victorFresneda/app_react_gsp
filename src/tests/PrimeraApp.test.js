import React from 'react';
import { shallow } from 'enzyme';
import PruebaApp from '../PrimeraApp';





describe('Pruebas en <PrimeraApp/>', () => {
    
    test('Debe mostrar <PrimeraApp/> correctamente ', () => {
    
        // const enviado = "Titulo enviado  desde los props";
        
        // const wrapper = render(<PruebaApp enviado={enviado}/>)

        const enviado = 'Titulo enviado desde los props'

        const wrapper = shallow( <PruebaApp  enviado={enviado}/>);

        expect( wrapper ).toMatchSnapshot();
    })
    
    
})
