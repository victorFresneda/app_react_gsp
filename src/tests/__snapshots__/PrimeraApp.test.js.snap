// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <PrimeraApp/> Debe mostrar <PrimeraApp/> correctamente  1`] = `
<Fragment>
  <h1>
     
    Titulo enviado desde los props
     
  </h1>
  <p>
     
    Parrafo desde default props
     
  </p>
</Fragment>
`;
